pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/copy/CopyExecutor.java

```diff
@@ -56,13 +60,32 @@ public void execute(Handle handle, long operatorId, long participantId, CopyConf
             CopyLocation target = pair.getTarget();
 
             if (source.getType() == CopyLocationType.ANSWER && target.getType() == CopyLocationType.ANSWER) {
+                QuestionDto sourceQuestion;
+                FormResponse sourceInstance;
                 String sourceStableId = ((CopyAnswerLocation) source).getQuestionStableId();
-                QuestionDto sourceQuestion = questionDtosByStableId.get(sourceStableId);
-                if (sourceQuestion == null) {
-                    continue; // Question might have been removed from activity, so we skip it.
+                String user = ((CopyAnswerLocation) source).getUser();
+                if (UserType.OPERATOR.equalsIgnoreCase(user)) {
+                    Stream<Governance> governances = handle.attach(UserGovernanceDao.class)
```

Comment:
Let's move the `attach()` call to get a `UserGovernanceDao` instance to outside of the scope of the parent `for` loop. This is an expensive operation, and it's not necessary to keep re-creating it.